sharktank/sharktank/examples/pipeline/export_ppffn_net.py

```diff
@@ -150,23 +163,26 @@ def main(raw_args=None):
             raise ValueError(
                 f"Parent directory for output file does not exist: {output_dir}"
             )
+    assert (
+        args.pipeline_parallelism_size > 1
+    ), "Pipeline parallelism size must be greater than 1."
 
     bs = 16
     sl = 128
-    primary_dim = 128 * 2**5
-    shard_count = 2
-    num_layers = 40
-    create_theta(primary_dim, shard_count, num_layers, save_path=args.output_irpa_file)
+    dim = 128 * 2**5
```

Comment:
nit: wouldnt hurt to provide context as to why you picked this, even if it isn't new to this patch